### ISSUE TITLE
Trouble installing dvc with pip: No matching distribution found for futures>=3.2.0 (from dvc)

### ISSUE THREAD

**Issue by sotte** [iterative/dvc#968]
I'm on a fresh ubuntu 18.04 and I want to install DVC. But I run into some dependency problems. Never had that problem before.
```
➤ virtualenv -p python3 .venv
➤ source .venv/bin/activate.fish
➤ pip install dvc
Collecting dvc
  Using cached https://files.pythonhosted.org/packages/d2/2d/117b6e99f4e7f0760d99944919d9dcaaeabfb6c6182a9c890b7260eec697/dvc-0.15.2-py2.py3-none-any.whl
Collecting pyasn1>=0.4.1 (from dvc)
  Using cached https://files.pythonhosted.org/packages/d1/a1/7790cc85db38daa874f6a2e6308131b9953feb1367f2ae2d1123bb93a9f5/pyasn1-0.4.4-py2.py3-none-any.whl
Collecting ply>=3.9 (from dvc)
  Using cached https://files.pythonhosted.org/packages/a3/58/35da89ee790598a0700ea49b2a66594140f44dec458c07e8e3d4979137fc/ply-3.11-py2.py3-none-any.whl
Collecting futures>=3.2.0 (from dvc)
  Could not find a version that satisfies the requirement futures>=3.2.0 (from dvc) (from versions: 0.2.python3, 0.1, 0.2, 1.0, 2.0, 2.1, 2.1.1, 2.1.2, 2.1.3, 2.1.4, 2.1.5, 2.1.6, 2.2.0, 3.0.0, 3.0.1, 3.0.2, 3.0.3, 3.0.4, 3.0.5, 3.1.0, 3.1.1)
No matching distribution found for futures>=3.2.0 (from dvc)
```
Here are all relevant version
```
➤ pip --version
pip 18.0 from /home/PATH/.venv/lib/python3.6/site-packages/pip (python 3.6)
➤ python --version
Python 3.6.5
➤ virtualenv --version
16.0.0
```

**Comment by efiop** [comment]
Hi @sotte !

Sorry, I accidentally pushed wrong version to pip. I released 0.15.3 already. Please upgrade.

**Comment by sotte** [comment]
That was quick. Thanks for the fix.

**Comment by efiop** [comment]
@sotte And thank you for the feedback! We really appreciate it! :slightly_smiling_face:

**Comment by Hong-Xiang** [comment]
I'm facing this problem again.
OS: Ubuntu (Mate) 16.04
Python: 3.6.6 (Anaconda)

```shell
$ pip install dvc
Collecting dvc
  Using cached 
.... (Some downloads)
Collecting futures>=3.2.0 (from dvc)
  Could not find a version that satisfies the requirement futures>=3.2.0 (from dvc) (from versions: 0.2.python3, 0.1, 0.2, 1.0, 2.0, 2.1, 2.1.1, 2.1.2, 2.1.3, 2.1.4, 2.1.5, 2.1.6, 2.2.0, 3.0.0, 3.0.1, 3.0.2, 3.0.3, 3.0.4, 3.0.5, 3.1.0, 3.1.1)
No matching distribution found for futures>=3.2.0 (from dvc)
```

```shell
$ pip search dvc
dvc (0.16.3)  - Git for data scientists - manage your code and data together
```

**Comment by efiop** [comment]
@Hong-Xiang Fix is released with dvc 0.16.5. Please feel free to try it out.
